start.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+chown -rf root:root ~/.ssh/config
+chmod -rf 644 ~/.ssh/config
```

Comment:
Can't this be set in the Dockerfile, so that it is run once, and not each time you want to use Gush?